doc-content/enterprise-only/weblogic/wls-configure-embedded-engine-proc.adoc

```diff
@@ -0,0 +1,286 @@
+  <artifactId>jbpm-kie-services</artifactId>
+</dependency>
+
+<dependency>
+  <groupId>org.jbpm</groupId>
+  <artifactId>jbpm-services-cdi</artifactId>
+</dependency>
+----
+endif::[]
++
+[id='_embedded_drools_engine_dependencies']
+.Embedded {DECISION_ENGINE} dependencies
+[source,xml]
+----
+<dependency>
+  <groupId>org.drools</groupId>
+  <artifactId>drools-compiler</artifactId>
+</dependency>
+
+<!-- Dependency for persistence support. -->
+<dependency>
+  <groupId>org.drools</groupId>
+  <artifactId>drools-persistence-jpa</artifactId>
+</dependency>
+
+<!-- Dependencies for decision tables, templates, and scorecards.
+For other assets, declare org.drools:business-central-models-* dependencies. -->
+<dependency>
+  <groupId>org.drools</groupId>
+  <artifactId>drools-decisiontables</artifactId>
+</dependency>
+<dependency>
+  <groupId>org.drools</groupId>
+  <artifactId>drools-templates</artifactId>
+</dependency>
+<dependency>
+  <groupId>org.drools</groupId>
+  <artifactId>drools-scorecards</artifactId>
+</dependency>
+
+<!-- Dependency for loading KJARs from a Maven repository using KieScanner. -->
+<dependency>
+  <groupId>org.kie</groupId>
+  <artifactId>kie-ci</artifactId>
+</dependency>
+
+----
++
+* To use the {KIE_SERVER}, declare the following dependencies:
++
+[id='_client_application_intelligent_process_server_dependencies']
+.Client application {KIE_SERVER} dependencies
+[source,xml]
+----
+<dependency>
+  <groupId>org.kie.server</groupId>
+  <artifactId>kie-server-client</artifactId>
+</dependency>
+
+----
+
+* To create a remote client for {PRODUCT}, declare the following dependency:
++
+.Client dependency
+[source,xml]
+----
+<dependency>
+  <groupId>org.uberfire</groupId>
+  <artifactId>uberfire-rest-client</artifactId>
+</dependency>
+----
++
+* When creating a JAR file that includes assets, such as rules and process definitions, specify the packaging type for your Maven project as `kjar` and use `org.kie:kie-maven-plugin` to process the `kjar` packaging type located under the `<project>` element. In the following example, `${kie.version}` is the Maven library version listed in
+ifdef::PAM[]
+https://access.redhat.com/solutions/3405361[What is the mapping between Red Hat Process Automation Manager and the Maven library version?]:
+endif::[]
+ifdef::DM[]
+https://access.redhat.com/solutions/3363991[What is the mapping between Red Hat Decision Manager and the Maven library version?]:
+endif::[]
++
+[source,xml]
+----
+<packaging>kjar</packaging>
+<build>
+ <plugins>
+  <plugin>
+   <groupId>org.kie</groupId>
+   <artifactId>kie-maven-plugin</artifactId>
+   <version>${kie.version}</version>
+   <extensions>true</extensions>
+  </plugin>
+ </plugins>
+</build>
+----
+
+
+ifdef::PAM[]
+. If the {PROCESS_ENGINE} or the {DECISION_ENGINE} along with persistence support is used in your project, you must declare the following hibernate dependencies in the `dependencyManagement` secion of your `pom.xml` file.
```

Comment:
If you use the ....with persistence support in your project, ...

Reply:
Ack